src/ui/public/notify/__tests__/notifier.js

```diff
@@ -185,6 +197,60 @@ describe('Notifier', function () {
     });
   });
 
+  describe('#custom', function () {
+    let customNotif;
+
+    beforeEach(() => {
+      customNotif = notifier.custom(customText, customParams);
+    });
+
+    afterEach(() => {
+      customNotif.clear();
+    });
+
+    it('has a custom function to make notifications', function () {
+      expect(notifier.custom).to.be.defined;
+    });
+
+    it('properly merges options', function () {
+      expect(customNotif).to.have.property('title', customParams.title);
+      expect(customNotif).to.have.property('lifetime', customParams.lifetime);
+
+      expect(customParams.title).to.be.a('string');
+      expect(customParams.lifetime).to.be.a('number');
```

Comment:
The goal of this test is to prove that the params are being assigned to the notification, right? If so, then I think these assertions can be simplified like this:

```javascript
expect(customNotif.title).to.equal(customParams.title);
expect(customNotif.lifetime).to.equal(customParams.lifetime);
```